### DISCUSSION TITLE
How to configure environment variables for 23R2 with Linux ?

### DISCUSSION THREAD

**Discussion by terryli1980** [ansys/pyaedt#4238] [Q&A] [answered]
Hi,

I am trying to develop some automation scripting with 23R2 in Linux.
There is a discussion about 22R2.
[Pyaedt v0.5.0 Released with Linux CPython Support](https://github.com/ansys/pyaedt/discussions/1366)

Could anyone show how to confiure for 23R2? Thanks!

BR,
Terry

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @terryli1980 ,

In Linux you need first to add these 2 environment variables (https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html#linux-support):

export ANSYSEM_ROOT222=/path/to/AedtRoot/AnsysEM/v222/Linux64
export LD_LIBRARY_PATH=$ANSYSEM_ROOT222/common/mono/Linux64/lib64:$ANSYSEM_ROOT222/Delcross:$LD_LIBRARY_PATH

Then, if you want to connect to an existing AEDT session, this session has to be opened in GRPC mode:
path\to\AnsysEM\v231\Lin64\ansysedt -grpcsrv 50352

https://aedt.docs.pyansys.com/version/stable/Getting_started/Troubleshooting.html
